src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx

```diff
@@ -11,6 +11,7 @@ import PopoverBase from '~shared/Extensions/PopoverBase';
 import TitledContent from '~common/Extensions/TitledContent/TitledContent';
 import WalletConnectedTopMenu from '../WalletConnectedTopMenu/WalletConnectedTopMenu';
 import Link from '~shared/Extensions/Link';
+import UserSubmenu from '../UserSubmenu/UserSubmenu';
```

Comment:
import UserSubmenu from '../UserSubmenu';

And use shared/common import prefix

Reply:
one `/UserSubmenu` removed but it's from partials folder so I think ../ isn't that bad